tests/Oryx.RuntimeImage.Tests/Node/NodeLegacyVersionsRuntimeImageTests.cs

```diff
@@ -91,6 +97,33 @@ public void NodeVersionMatchesImageName(string nodeTag, string nodeVersion)
                 result.GetDebugInfo());
         }
 
+        [Theory]
+        [InlineData("10")]
+        [InlineData("10.1")]
+        [InlineData("10.10")]
+        [InlineData("10.12")]
+        [InlineData("10.14")]
+        public void Node10ImageContains_Correct_NPM_Version(string imageTag)
```

Comment:
Do we still need this if we don't support node 10 anymore?

Reply:
we still release 10, so technically we do need to test them to some extent, but we wont do extensive testing any more